packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -30,81 +35,59 @@ class InitCommand extends ShorebirdCommand
       return ExitCode.noUser.code;
     }
 
-    final progress = logger.progress('Initializing Shorebird');
     try {
       if (!hasPubspecYaml) {
-        progress.fail('''
+        logger.err('''
 Could not find a "pubspec.yaml".
 Please make sure you are running "shorebird init" from the root of your Flutter project.
 ''');
         return ExitCode.noInput.code;
       }
     } catch (error) {
-      progress.fail('Error parsing "pubspec.yaml": $error');
+      logger.err('Error parsing "pubspec.yaml": $error');
       return ExitCode.software.code;
     }
 
-    final ShorebirdYaml? shorebirdYaml;
-    try {
-      shorebirdYaml = getShorebirdYaml();
-    } catch (_) {
-      progress.fail('Error parsing "shorebird.yaml".');
+    if (hasShorebirdYaml) {
+      logger.err('''
+"shorebird.yaml" already exists.
```

Comment:
This assumes that the user is running init intentionally. We might want to add language here to the tune of "you're already set up to use shorebird" to tell the user that this step has been completed.

Reply:
Updated the wording and also included a `--force` flag if you want to reinitialize despite an existing `shorebird.yaml`